html/browsers/the-window-object/close-method.window.js

```diff
@@ -0,0 +1,30 @@
+async_test(t => {
+  const openee = window.open();
+  assert_false(openee.closed);
+  openee.close();
+  assert_true(openee.closed);
+  assert_equals(openee.opener, self);
+  t.step_timeout(() => {
+    assert_equals(openee.opener, null);
```

Comment:
Again doesn't hurt to also check .closed here. Basically always check them together.